fastlane/Fastfile

```diff
@@ -555,6 +555,29 @@ platform :ios do
       sh 'cp -R ../dist/assets/release/splash_screen/ios/SplashBackground.imageset ../ios/Mattermost/Images.xcassets/'
       sh 'cp -R ../dist/assets/release/splash_screen/ios/SplashIcon.imageset ../ios/Mattermost/Images.xcassets/'
     end
+
+    app_name =  ENV['APP_NAME'] || 'Mattermost'
```

Comment:
```suggestion
    app_name =  ENV['APP_NAME'] || 'Mattermost Beta'
```

`Mattermost Beta` should be the default if the env var is not set, same as in other places

Reply:
> Mattermost Beta should be the default if the env var is not set, same as in other places

Yeah, right. The only reason why I set 'Mattermost' is because $(PRODUCT_NAME) is always replaced with that value.